packages/ui/form/PhoneInput.tsx

```diff
@@ -18,11 +18,19 @@ export type PhoneInputProps = {
 };
 
 function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputProps) {
+  useEffect(() => {
+    if (!rest.value) {
+      return
+    }
+    const formattedValue = rest.value.trim().replace(/^\+?/, "+") 
+    onChange(formattedValue);
+  }, []);
```

Comment:
Have done this because it was giving `Invalid phone number` whenever I try to submit it. Other wise it can be handled via input fields too